_rules/form-field-label-descriptive-cc0f0a.md

```diff
@@ -243,8 +246,9 @@ The `label` element is not [visible][].
 
 ```html
 <html lang="en">
-	<label for="fname" style="display:none;">First name:</label>
-	<input id="fname" type="text" name="fname" />
+	<label style="display:none;">
+		First name: <input type="text" />
+	</label>
```

Comment:
This makes the `input` not visible. The Applicability is that both the label and the form field are visibles, so I'd rather keep an Inapplicable test case where one is visible but not the other (so we can check that implementers do indeed test both visibility and not just the form field visibility).

Reply:
We have the visible label, hidden control case covered by inapplicable example 3. But I don't think we can have a hidden label, visible control case without it being a failure of SC 3.3.2. I don't really think we should have things like this as inapplicable examples.

Reply:
The rule does not map to 3.3.2, so it's not a problem (with respect to the rule format and our design document) to have such an Inapplicable example.

---
We can also make the example something like
```html
<label for="foo" class="sr-only">Foo</label>
<span>Foo:</span><input id="foo"/>
```
This would pass 3.3.2 since the `span` does serve as a WCAG "label", but would have a visible form field with an invisible programmatic label and thus be Inapplicable for the rule.